### DISCUSSION TITLE
Error occurs when fitting the model in: Estimating the effective reproduction number in real-time for a single timeseries with reporting delays

### DISCUSSION THREAD

**Discussion by christinesangphet** [epinowcast/epinowcast#492] [Q&A]
I'm following the vignette "Estimating the effective reproduction number in real-time for a single timeseries with reporting delays" and everything runs fine until I reach the "Fitting the model" section. When attempting to produce germany_nowcast and retro_germany_nowcast, I receive the following error:

Error in `.fn()`:
! The generation time must sum to 1
Backtrace:
 1. epinowcast::epinowcast(...)
 2. global expectation_module(data = rt_germany_pobs)
 3. epinowcast (local) .fn(...)

The vignette also displays a different error when running the same code:

#> Error in expectation_module(data = rt_germany_pobs): object 'rt_formula' not found

However, it does not provide a resolution for this error. I'm unsure if these are related issues.

**Comment by seabbs** [comment]
Hi @christinesangphet, thanks for opening this! 

I've just checked this locally and everything is running as expected as far as I can see. It is possible an error has crept in as we use precompiled vignettes (i.e we manually update them) so they can fall out of step with the code base (opening an issue now to make checking this better) but given its working locally for me I think that isn't the case here. 

I think this could be two things: 

1. A dependency issue - I would make sure you have all your dependencies up to date and in particular `epinowcast` (try installing it from Github). 
2. A stochastic issue - this could maybe be a strange edge case caused by a random number draw. I suggest rerunning the vignette in a clean R session to check. 

For now I am porting this to a Q and A but will open issues as and when we find problems! 

Thanks, 

Sam

Edit: See #493 for actions taken to make these checks less manual/more robust.
